modules/highgui/include/opencv2/highgui.hpp

```diff
@@ -496,7 +524,7 @@ The function sets the maximum position of the specified trackbar in the specifie
 
 @note
 
-**[Qt Backend Only]** winname can be empty (or NULL) if the trackbar is attached to the control
+<b>[Qt Backend Only]</b> winname can be empty (or NULL) if the trackbar is attached to the control
```

Comment:
Just checked: it should be `[__Qt Backend Only__]`, because emphasis does not work when phrase starts with the bracket ([doxygen specific markdown](https://www.stack.nl/~dimitri/doxygen/manual/markdown.html#mddox_emph_spans)).